munch-match/src/services/MapsService.js

```diff
@@ -1,118 +1,129 @@
-
 /**
- * This script interacts with the google maps places api to retrieve the necessary information. 
- * 
+ * This script interacts with the google maps places api to retrieve the necessary information.
+ *
  * getResults() - Takes input the query type and the users coordinate information to retrieve information such as place name, rating etc..
- * fetchAllDetails() - 
+ * fetchAllDetails() -
  */
 
 
 //Creates the google map variable
 const google = window.google;
 
-function getResults(queryType, lat, lng) {
-  //Stores the user's location as google.maps.LatLng (Latitude and Longitude)
-  //Change location variables to props variables
-  const userLocation = new google.maps.LatLng(lat, lng);
+function getResults(queryType, lat, lng, ratingRange, priceRanges) {
```

Comment:
A small improvement here would be to validate the inputs (I know priceRange and ratingRange are sliders but it would still be beneficial and make the code more robust i think:
so something like?
`function getResults(queryType, lat, lng, ratingRange, priceRanges) {
    if (!queryType || !lat || !lng || !ratingRange || !priceRanges) {
        throw new Error('All input parameters must be provided.');
    }
    const userLocation = new google.maps.LatLng(lat, lng);
    // ...
}`